py3status/modules/sway_idle.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display sway inhibit idle status.
+
+This Module shows an indicator, if an idle is inhibited by an inhibitor.
+For more information about inhibit idle see `man 5 sway`
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default 1)
+    format: Display format (default 'Inhibit Idle: [\?if=inhibit_idle=True True]|False')
+
+Format placeholders:
+    {inhibit_idle} Returns 'True' if idle is inhibited, 'False' else.
+
+Example:
+
+```
+sway_idle {
+    format = "[\?if=inhibit_idle=True Idle Inhibited]"
+}
+```
+
+@author Valentin Weber <valentin+py3status@wv2.ch>
+@license BSD
+
+SAMPLE OUTPUT
+{full_text': 'Inhibit Idle: True'}
+"""
+
+
+class Py3status:
+
+    # available configuration parameters
+    cache_timeout = 1
+    format = "Inhibit Idle: [\?if=inhibit_idle=True True]|False"
+
+    def sway_idle(self):
+        sway_tree = self.py3.command_output(self.py3.get_wm_msg() + " -t get_tree")
+        inhibit_idle = str(sway_tree.find('"inhibit_idle": true') > 0)
```

Comment:
We probably want to preserve this one as Boolean. Using pseudo '"idle is true" `in` tree' might be enough.

Reply:
Yes, I rewrote the string search.